sct.py

```diff
@@ -369,6 +369,8 @@ def conf(config_file, backend):
 def conf_docs(output_format):
     add_file_logger()
 
+    os.environ['SCT_CLUSTER_BACKEND'] = "aws"  # just to pass SCTConfiguration() verification.
```

Comment:
you still need it ? I guess this is since AWS was removed from the default in the main command

Reply:
Yep, we need it. Looks like nobody ran this command for a long time.